### ISSUE TITLE
redis hash error while installing

### ISSUE THREAD

**Issue by RayyanNafees** [ScoopInstaller/Main#4894]
### Prerequisites

- [X] I have used the predefined issue title. (e.g. "xxx@xxx&#58; hash check failed")
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name and Version

redis@7.0.1

**Comment by RayyanNafees** [comment]
Here is the terminal output
![image](https://github.com/ScoopInstaller/Main/assets/59685062/78f2a3f7-81b6-4c5f-8023-fa31589cab99)

**Comment by cnbattle** [comment]
Fixed  https://github.com/ScoopInstaller/Main/commit/41eeb77c905919c74b27d4ec6b256fa8a0eab4f9

**Comment by cnbattle** [comment]
use `scoop update`
```shell
PS D:\> scoop update
Updating Scoop...
Updating 'main' bucket...
 * 41eeb77c9 redis@7.0.11: Fix hash (Closes #4896)                       3 minutes ago
 * 8de1a73ec supabase: Add version 1.68.6 (#4890)                        6 minutes ago
Scoop was updated successfully!
```
